includes/definitions/discovery/snr-erd.yaml

```diff
@@ -8,13 +8,14 @@ modules:
                  - diAlarmName
                  - doName
                  - doDeviceName
+                 - rsshtpName
     sensors:
         temperature:
             data:
                 -
-                    oid: sysTemperature
```

Comment:
So was this wrong all along?

Reply:
It was temporary, the sensor was delivered with a delay, like the mib to it